test/controllers/update/update.continue.saved.filing.controller.spec.ts

```diff
@@ -74,6 +87,19 @@ describe("Continue with saved filing controller", () => {
       expect(resp.text).toContain(CONTINUE_SAVED_FILING_PAGE_TITLE);
       expect(resp.text).toContain(PAGE_TITLE_ERROR);
       expect(resp.text).toContain(ErrorMessages.UPDATE_SELECT_IF_CONTINUE_SAVED_FILING);
+      expect(resp.text).toContain(UPDATE_SERVICE_NAME);
     });
+
+    // TODO Uncomment this test (which should pass) when a solution to handling validation on the 'remove' journey is implemented
```

Comment:
Left this test here. It was running and passing but then I had to rollback the validation middleware change I made as it broke something else. Might as well leave it as will be needed some point soon.